web/beacon-app/src/features/projects/components/Settings.tsx

```diff
@@ -21,18 +34,29 @@ const ProjectSettings = () => {
   return (
     <>
       <div>
-        <Button variant="ghost" className="flex-end bg-inherit w-8 border-none" onClick={open}>
-          <div className="ml-[4.5rem]">
-            <SettingIcon />
-          </div>
+        <Button
+          variant="ghost"
+          size="custom"
+          className="flex-end bg-inherit hover:bg-transparent border-none"
+          onClick={open}
+        >
+          <SettingIcon />
         </Button>
         <Menu open={isOpen} onClose={close} anchorEl={anchorEl}>
           <Menu.Item onClick={openDeleteModal} data-testid="cancelButton">
             <Trans>Delete Project</Trans>
           </Menu.Item>
+          <Menu.Item onClick={openRenameModal} data-testid="rename-project">
+            <Trans>Rename Project</Trans>
```

Comment:
`Rename Project` is currently only listed on the `Projects` page in the Figma designs. The `Project Details` page only gives users an option to `Delete Project` when the cogwheel is clicked. Should this be updated or has a changed been discussed but not added to the Figma designs?